doc/man3/SSL_CTX_set_alpn_select_cb.pod

```diff
@@ -21,13 +22,21 @@ protocol negotiation (ALPN)
                                             const unsigned char *in,
                                             unsigned int inlen,
                                             void *arg), void *arg);
+ void SSL_CTX_set_next_protos_advertised_cb(SSL_CTX *ctx,
+                                   SSL_CTX_npn_advertised_cb_func cb,
+                                   void *arg);
+ void SSL_CTX_set_next_proto_select_cb(SSL_CTX *ctx,
+                               SSL_CTX_npn_select_cb_func cb,
```

Comment:
I don't see the SSL_CTX_npn_select_cb_func anywhere, and so talking about parameters you give to that is hard to follow.